src/us/kbase/workspace/test/workspace/ReferenceTest.java

```diff
@@ -32,13 +32,14 @@ public void mongoRef() throws Exception {
 	public void failRefStringConst() throws Exception {
 		failMakeRef(null, "reference cannot be null or the empty string");
 		failMakeRef("", "reference cannot be null or the empty string");
-		failMakeRef("1", "Illegal number of separators / in object reference 1");
+		failMakeRef("1", "Illegal number of separators '/' in object reference '1'");
 		failMakeRef("1/2", "ref 1/2 is not an absolute reference");
 		failMakeRef("1/foo", "ref 1/foo is not an absolute reference");
 		failMakeRef("foo/1/2", "ref foo/1/2 is not an absolute reference");
 		failMakeRef("1/foo/2", "ref 1/foo/2 is not an absolute reference");
-		failMakeRef("1/1/foo", "Unable to parse version portion of object reference 1/1/foo to an integer");
-		failMakeRef("1/2/3/4", "Illegal number of separators / in object reference 1/2/3/4");
+		failMakeRef("1/1/foo",
+				"Unable to parse version portion of object reference 1/1/foo to an integer");
```

Comment:
no `'` around the object reference here?

Reply:
Done. Didn't actually touch the code that throws this error, but yeah, should be consistent